src/modules/profile/views/waitingTimeStatistics/waitingTimeStatistics.tsx

```diff
@@ -1,14 +1,12 @@
-import { ArcElement, Chart as ChartJS, type ChartOptions, Legend, Tooltip } from 'chart.js';
-import { useEffect, useMemo, useState } from 'react';
-import { Pie } from 'react-chartjs-2';
-
-ChartJS.register(ArcElement, Tooltip, Legend);
+import { useMemo } from 'react';
+import WaitingTimeChart from './waitingTimeChart';
+import _ from 'lodash';
```

Comment:
If you just want to include `sortBy` not the rest of the lodash functions (useful for keeping your bundle size small), you can do this

`import sortBy from 'lodash/sortBy'`